frb_codegen/src/ir/func.rs

```diff
@@ -19,7 +20,7 @@ impl IrFunc {
 /// Represents a function's output type
 #[derive(Debug, Clone)]
 pub enum IrFuncOutput {
-    ResultType(IrType),
+    ResultType(IrType, Option<IrType>),
```

Comment:
Well, shall we make it at least named fields, s.t. it is clearer what the two fields mean